### PR TITLE
SFD-3847 Sonar code quality fixes included

### PR DESCRIPTION
**Pull request by ArifNhs**
SFD 3847 sonar code quality fixes included to include lombok and ignore third party library files

### CODE REVIEW

File: .github/workflows/code-quality.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@ jobs:
           args: >
             -Dsonar.organization=nhsd-exeter
             -Dsonar.projectKey=uec-dos-api-sfsa
-            -Dsonar.java.binaries=target/classes
+            -Dsonar.java.binaries=target/classes -Dsonar.java.libraries=target/lib/*.jar
```
Comment (gwaardenburg) [inline .github/workflows/code-quality.yml:36]: can we pop this onto a new line please just to keep it readable

Comment (ArifNhs) [reply .github/workflows/code-quality.yml]: done

**Review by gwaardenburg — APPROVED** [review]

File: application/app/src/main/java/uk/nhs/digital/uec/api/authentication/localstub/LocalConstants.java [hunk 2]
```diff
@@ -10,7 +10,7 @@ private LocalConstants() {}
 
   public static final String KEY_STORE = "integTest.jks";
 
-  public static final String KEY_STORE_PASSWORD = "mypass";
```
Comment (gwaardenburg) [inline application/app/src/main/java/uk/nhs/digital/uec/api/authentication/localstub/LocalConstants.java:13]: I wonder if we can add these local files to be ignored by the git password hook...

File: application/app/src/main/java/uk/nhs/digital/uec/api/authentication/localstub/LocalJwtFactory.java [hunk 3]
```diff
@@ -24,7 +24,7 @@ public String createToken(
     Date nowDate = new Date(nowMs);
 
     KeyStoreKeyFactory keyStoreKeyFactory =
```
Comment (gwaardenburg) [inline application/app/src/main/java/uk/nhs/digital/uec/api/authentication/localstub/LocalJwtFactory.java:26]: I wonder if we can add these local files to be ignored by the git password hook...

**Review by gwaardenburg — APPROVED** [review]

File: application/app/src/main/java/uk/nhs/digital/uec/api/authentication/localstub/LocalJwtFactory.java (hunk 3 above)
Comment (ArifNhs) [reply application/app/src/main/java/uk/nhs/digital/uec/api/authentication/localstub/LocalJwtFactory.java:26]: git secret -scan doesn't have option to ignore files but we can include files with this option

**Review by daonhs — APPROVED** [review]

**Pull request merged**